packages/jest-dom-mocks/package.json

```diff
@@ -29,6 +32,8 @@
     "lolex": "^2.3.2"
   },
   "devDependencies": {
+    "eslint": "^4.19.1",
+    "eslint-plugin-shopify": "^22.0.0",
```

Comment:
why 22 here, but 20 everywhere else?

Reply:
Should be 22 everywhere prob.

Reply:
I'll make it 20 now and then we can fix it for all of them.